nengo/neurons.py

```diff
@@ -42,18 +59,21 @@ def gain_bias(self, max_rates, intercepts):
 
         Parameters
         ----------
-        max_rates : ndarray(dtype=float64)
+        max_rates : (n_neurons,) array_like
             Maximum firing rates of neurons.
-        intercepts : ndarray(dtype=float64)
+        intercepts : (n_neurons,) array_like
             X-intercepts of neurons.
 
         Returns
         -------
-        gain : ndarray(dtype=float64)
+        gain : (n_neurons,) array_like
             Gain associated with each neuron. Sometimes denoted alpha.
-        bias : ndarray(dtype=float64)
+        bias : (n_neurons,) array_like
             Bias current associated with each neuron.
         """
+        max_rates = np.array(max_rates, dtype=float, copy=False, ndmin=1)
```

Comment:
Yep, that's right. I don't know if the `ndmin` is super necessary, but @hunse included it in his PR so I trust he did it for a reason!